res/css/views/rooms/_EventTile.pcss

```diff
@@ -744,6 +744,10 @@ $left-gutter: 64px;
                 > * {
                     display: inline;
                 }
+
+                .mx_Emoji {
+                    font-size: inherit; /* set the same font-size to emoji inside a code block */
```

Comment:
For the record: this seems to be overriding a setting in `_RichText.pcss`.

I wonder if a better solution would be to add a `:not:(pre > *)` selector to the `.mx_Emoji`  class. What do you think?